Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM amazonlinux:2023 AS base
 ENV NODE_VERSION=24.4.0
 
 RUN yum update -y && \
-    yum install -y tar xz openssl && \
+    yum install -y tar xz openssl curl-minimal && \
```

Comment:
You'll need to double check me on this, but I believe the amazonlinux:2023 image contains `curl` already.